File: 1/One.java
/**
 * Solves Project Euler problem #1
 * https://projecteuler.net/problem=1
 * 
 * @author Jake Conway
 * @version 1.0
 */
 
public class One
{
    public static void main(String[] args)
    {
        int total = 0;
        for(int i = 0; i < 1000; i++)
        {
            if(i%3 == 0 || i%5 == 0)
            {
                total += i;
            }
        }
        System.out.println("Solution: " + total);
    }
}

File: Solutions/1-9/1/one.py
class One:
  @staticmethod
  def main():
    total = 0;
    for x in range(0,1000):
      if x%3 == 0 or x%5 == 0:
        total += x
    print("Solution: " + str(total))
if __name__ == '__main__':
  One.main()
